nntrainer/layers/layer_node.cpp

```diff
@@ -18,13 +18,17 @@
 #include <nntrainer_error.h>
 #include <nntrainer_log.h>
 #include <node_exporter.h>
+#include <profiler.h>
 #include <time_dist.h>
 #include <util_func.h>
 
 #include <base_properties.h>
 #include <common_properties.h>
 
 namespace nntrainer {
+static constexpr const char *FORWARD_SUFFIX = ":forward";
```

Comment:
how about using function name instead of predefined strings?

Reply:
you mean __func__? registering point doesn't have access to actual func name (for performance concern) where it is executed, so we will need a predefined strings.

Reply:
alright.